pkg/apis/ceph.rook.io/v1/cleanup.go

```diff
@@ -17,6 +17,18 @@ limitations under the License.
 package v1
 
 const (
+	// SanitizeDataSourceZero uses /dev/zero as sanitize source
+	SanitizeDataSourceZero SanitizeDataSourceProperty = "zero"
+
+	// SanitizeDataSourceRandom uses /dev/random as entropy source
```

Comment:
By default, `shread` uses not /dev/random but ISAAC.

https://github.com/coreutils/coreutils/blob/master/gl/lib/randread.c#L299

However, it's not necessary to describe the entropy source since it's not written in the document. 

```suggestion
	// SanitizeDataSourceRandom uses an entropy source that `shred` command selects by default.
```

Or "... uses random bytes as sanitize source" is also OK.

Reply:
Thanks for verifying!